### PR TITLE
add usage with filter decorator info to README.md

### LINKED ISSUE #10: Add documentation how to combine with `filterState`
I started out with this:

```
const engine = migrate(
  debounceSaves(
    filterState(createEngine('my-save-key'), […]),
    saveDelay
  ),
  currentStateVersion
)
```

I was initially bugged out that `migrate` seemed to always apply all migrations. It took me a bit to suspect that my `filterState` application was filtering out the version key. I confirmed that looking at the source code of decorator-migrate.

It would be helpful to add instructions that either 
1. `filterState` should be applied after `migrate` – or
2. `filterState` should include the version key.

I went for the second option in order to make it more explicit what is going on:

```
const stateVersionProp = '_stateVersion'

const engine = migrate(
  debounceSaves(
    filterState(createEngine('my-save-key'), […, stateVersionProp]),
    saveDelay
  ),
  currentStateVersion,
  stateVersionProp
)
```

### PR DESCRIPTION
**Pull request by axe312ger**
closes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,33 @@ engine.addMigration(2, (state) => { /* migration step for 2 */ return state; });
...
+```js
+import * as storage from 'redux-storage'
+import createEngine from 'redux-storage-engine-localstorage'
+import filter from 'redux-storage-decorator-filter'
+import debounce from 'redux-storage-decorator-debounce'
+import migrate from 'redux-storage-decorator-migrate'
+
+const reducer = storage.reducer(reducers)
+const stateVersionProp = '_stateVersion'
+const whitelist = [stateVersionProp]
```
Comment (mathieudutour) [inline README.md]: Can you add a comment here to bring attention to this line?

**Comment by mathieudutour** [comment]
awesome, will merge after the small change needed

**Comment by axe312ger** [comment]
Done :)

**Comment by mathieudutour** [comment]
thanks!

**Pull request merged**